tests/components/nasweb/conftest.py

```diff
@@ -0,0 +1,64 @@
+"""Common fixtures for the NASweb tests."""
+from collections.abc import Generator
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
+
+from homeassistant.components.nasweb.coordinator import NotificationCoordinator
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.nasweb.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+BASE = "homeassistant.components.nasweb.config_flow"
+WEBIO_API_CHECK_CONNECTION = f"{BASE}.WebioAPI.check_connection"
+WEBIO_API_REFRESH_DEVICE_INFO = f"{BASE}.WebioAPI.refresh_device_info"
+INITIALIZE_NOTIFICATION_COORDINATOR = f"{BASE}.initialize_notification_coordinator"
+WEBIO_API_GET_SERIAL_NUMBER = f"{BASE}.WebioAPI.get_serial_number"
+GET_HASS_ADDRESS_FROM_ENTRY = f"{BASE}.get_hass_address_from_entry"
+WEBIO_API_STATUS_SUBSCRIPTION = f"{BASE}.WebioAPI.status_subscription"
+NOTIFICATION_COORDINATOR_CHECK_CONNECTION = (
```

Comment:
I find these a bit hard to read.

Maybe do like this to make it clearer what's patched?
```py
    with patch(
        BASE + "WebioAPI.check_connection",
        return_value=True,
    ) as check_connection
```